descriptors/ModuleDescriptor-template.json

```diff
@@ -2,6 +2,31 @@
   "id": "${artifactId}-${version}",
   "name": "Circulation Module",
   "provides": [
+    {
+      "id": "requests-report",
```

Comment:
Can this and the path be made plural, just in case we add additional custom reports. This might avoid a breaking change later.

Reply:
yes, we can

Reply:
Updated